utils/ai/context/index.ts

```
@@ -34,9 +34,7 @@ async function fetchHistory(
       dictionary.registerUser(msg.member || msg.author);
 
       msg.mentions.users.forEach((u) => dictionary.registerUser(u));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)